aio/tools/transforms/templates/api/lib/descendants.html

```
@@ -4,7 +4,7 @@
   {% for descendant in descendants %}
     <li>
       <code>{$ descendant.name $}</code>
-      {$ renderDescendantList(descendant.descendants | filterByPropertyValue('docType', docTypeMatcher), docType, recursed, docTypeMatcher) $}
+      {$ renderDescendantList(descendant.descendants | filterByPropertyValue('docType', docTypeMatcher) | filterByPropertyValue('privateExport', undefined), docType, recursed, docTypeMatcher) $}
```

Could we also make [this][1] a hard failure (instead of a warning) now? :pray:

[1]: https://github.com/angular/angular/blob/b48eabddb83900496e5e00fb98e192c6c49dff26/aio/tools/transforms/angular-base-package/post-processors/auto-link-code.js#L117-L118

---

Adding

---

Done - https://github.com/angular/angular/pull/40404/commits/939d58a31084b58fddbad57396efd5d3a058ed56